### ISSUE TITLE
ACapi: Log result of initial API call to Datadog

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-team#99760]
### User story

As a Cartography team member, I want the landing page to log the results of all ACapi calls to Datadog, so that we can compare to what Users are actually experiencing (per GA).

### Notes
- Broken out from #99629, which will log subsequent re-try

Possible tasks:
- 

## Acceptance criteria

- Successes and failures are visible in an appropriate MHV-on-VA.gov dashboard, along with error codes
- QA review

**Comment by mcarterA6** [comment]
Work is being moved to #99870.

**Comment by wesrowe** [comment]
No-op for now, Daniel added a widget to the dash that seems to meet the need. We may revisit later.

**Comment by wesrowe** [comment]
Kevin posted a useful DD query [in slack](https://dsva.slack.com/archives/C0581MN69TJ/p1736795804441159?thread_ts=1736793216.671949&cid=C0581MN69TJ):

>`env:eks-prod "[User][MHVUserAccountsController] show error" @payload.errors.code:805`

**Comment by dcloud** [comment]
May already exist in https://app.ddog-gov.com/sb/f327ad72-c02a-11ec-a50a-da7ad0900007-99a8a148d0970a6db4d3764fa7fdefbf (or that may need updating/tweaking)

**Comment by kjsuarez** [comment]
Report added to our DD dashboard
